hadoop-tools/hadoop-aws/src/main/java/org/apache/hadoop/fs/s3a/DefaultS3ClientFactory.java

```diff
@@ -165,14 +153,41 @@ public S3AsyncClient createS3AsyncClient(
             configureClientBuilder(S3AsyncClient.builder(), parameters, conf, bucket)
                 .httpClientBuilder(httpClientBuilder);
 
-    // multipart upload pending with HADOOP-19326.
```

Comment:
safe to cut this now as SDK no longer complains with  `Multipart download is not yet supported. Instead use the CRT based S3 client for multipart download.`  when trying to do a Ranged GET with the S3 Async client and with `>2.29.52`